public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -210,14 +171,11 @@ export class SharedPreferences extends PureComponent<Props, State> {
                 }
                 data-testid="User preferences home dashboard drop down"
               >
-                <Select
-                  value={dashboards.find((dashboard) => dashboard.uid === homeDashboardUID)}
-                  getOptionValue={(i) => i.uid}
-                  getOptionLabel={this.getFullDashName}
-                  onChange={(dashboard: SelectableValue<DashboardSearchItem>) =>
-                    this.onHomeDashboardChanged(dashboard.uid)
-                  }
-                  options={dashboards}
+                <DashboardPicker
```

Comment:
We probably can remove the hint. It doesn't make sense anymore

<img width="760" alt="Captura de Pantalla 2022-08-26 a las 10 04 13" src="https://user-images.githubusercontent.com/5699976/186853895-12cfa84f-b975-434f-8b09-ee98168b06db.png">

Reply:
done